spec/javascripts/packages/document-capture/components/document-capture-spec.jsx

```diff
@@ -275,4 +294,106 @@ describe('document-capture/components/document-capture', () => {
     const hasValueSelected = !!getByLabelText('doc_auth.headings.document_capture_front');
     expect(hasValueSelected).to.be.true();
   });
+
+  it('renders async upload pending progress', async () => {
+    const statusChecks = 3;
+    let remainingStatusChecks = statusChecks;
+    sandbox.stub(window, 'fetch');
+    const upload = sinon.stub().callsFake((payload, { endpoint }) => {
+      switch (endpoint) {
+        case 'about:blank#upload':
+          expect(payload).to.have.keys([
+            'front_image_iv',
+            'front_image_url',
+            'back_image_iv',
+            'back_image_url',
+            'selfie_image_iv',
+            'selfie_image_url',
+          ]);
+
+          return Promise.resolve({ success: true, isPending: true });
+        case 'about:blank#status':
+          expect(payload).to.be.empty();
+
+          return Promise.resolve({ success: true, isPending: Boolean(remainingStatusChecks--) });
+        default:
+          throw new Error();
+      }
+    });
+    const key = await window.crypto.subtle.generateKey(
+      {
+        name: 'AES-GCM',
+        length: 256,
+      },
+      true,
+      ['encrypt', 'decrypt'],
+    );
+
+    const { getByLabelText, getByText, getAllByText, findAllByText } = baseRender(
+      <UploadContextProvider
+        endpoint="about:blank#upload"
+        statusEndpoint="about:blank#status"
+        backgroundUploadURLs={{
+          front: 'about:blank#front',
+          back: 'about:blank#back',
+          selfie: 'about:blank#back',
```

Comment:
should this be `#selfie` for consistency?
```suggestion
          selfie: 'about:blank#selfie',
```

Reply:
Oops. Yes 😄